addons/Makefile

```diff
@@ -118,7 +118,8 @@ docker-build-and-publish: docker-build docker-publish kbld-image-replace
 
 .PHONY: kbld-image-replace
 kbld-image-replace: ## Add newImage in kbld-config.yaml
-	cd ../hack/packages/kbld-image-replace && $(MAKE) run IMAGE=$(IMG_DEFAULT_NAME_TAG) NEW_IMAGE=$(IMG)
+	cd ../hack/packages/kbld-image-replace && \
+	  go run main.go -kbld-config ../../../packages/management/addons-manager-v2/kbld-config.yaml $(IMG_DEFAULT_NAME_TAG) $(IMG)
```

Comment:
@shyaamsn I believe you've changed ```addons-manager-v2``` to ```addons-manager``` in [this PR](https://github.com/vmware-tanzu/tanzu-framework/pull/2315). Should the path ```../../../packages/management/addons-manager-v2/kbld-config.yaml```  here get changed to ```../../../packages/management/addons-manager/kbld-config.yaml``` ?

Reply:
Thanks for the info. Updated.